docs/index.asciidoc

```diff
@@ -72,6 +72,9 @@ input plugins.
 
 Labels for facility levels. These are defined in RFC3164.
 
+You can use this option to override the integer->label mapping for syslog inputs
+that behave differently than the RFCs. 
```

Comment:
When a user overrides this, they need to provide a zero-indexed array with _all_ of their facility labels in order; when a log message arrives with a facility number with no corresponding entry, the `facility_label` is not added to the event.

The same is true for `severity_labels` option below, so we should add the note there too.

Reply:
Thanks for the info.  I hope that I interpreted your comments correctly. 
If not, I know I can count on you to keep me honest. :-) As always, I appreciate your help.